src/classes/RD_RecurringDonations.cls

```diff
@@ -803,73 +804,79 @@ public class RD_RecurringDonations {
-            if (r.npe03__Open_Ended_Status__c == closedlabel && oldRD.npe03__Open_Ended_Status__c != r.npe03__Open_Ended_Status__c){
+            if (r.npe03__Open_Ended_Status__c == closedlabel && oldRD.npe03__Open_Ended_Status__c != r.npe03__Open_Ended_Status__c) {
                 newlycloseddonations.add(r);
             }
 
             //if there's any other change that would require a reevaluation of existing Opps
-            else if (r.npe03__Open_Ended_Status__c != closedlabel){
+            else if (r.npe03__Open_Ended_Status__c != closedlabel) {
                 // for non open-ended rd's, just always re-insert
                 if (r.npe03__Open_Ended_Status__c != openlabel) {
                     reinsertopps.add(r);
-                }
-                else if (r.npe03__Amount__c != oldRD.npe03__Amount__c ||
+
+                } else if (r.npe03__Amount__c != oldRD.npe03__Amount__c ||
                     r.npe03__Recurring_Donation_Campaign__c != oldRD.npe03__Recurring_Donation_Campaign__c ||
                     r.npe03__Organization__c != oldRD.npe03__Organization__c ||
                     r.npe03__Next_Payment_Date__c != oldRD.npe03__Next_Payment_Date__c ||
                     r.Always_Use_Last_Day_Of_Month__c != oldRD.Always_Use_Last_Day_Of_Month__c ||
                     cfmFieldChanged==true
-                    ){
+                ) {
                     reevaluateopps.add(r.id);
-                }
-                else if (r.npe03__Installment_Period__c != oldRD.npe03__Installment_Period__c || r.npe03__Contact__c != oldRD.npe03__Contact__c){
+
+                } else if (r.npe03__Installment_Period__c != oldRD.npe03__Installment_Period__c || r.npe03__Contact__c != oldRD.npe03__Contact__c) {
                     reinsertopps.add(r);
                 }
             }
         }
 
-        if (!newlycloseddonations.isEmpty()){
+        if (!newlycloseddonations.isEmpty()) {
             runRecurringDonationClosedActions(newlycloseddonations, dmlWrapper);
         }
-        if (!reevaluateopps.isEmpty()){
-            if (reevaluateopps.size() == 1 || system.isFuture() || system.isBatch())
+
+        if (!reevaluateopps.isEmpty()) {
+            if (reevaluateopps.size() == 1 || System.isFuture() || System.isBatch()) {
                 updateExistingOpps(reevaluateopps, dmlWrapper);
-            else
+            } else {
                 updateExistingOppsFuture(reevaluateopps);
+            }
         }
+
         //needs a total reinsert, not an update
-        if (!reinsertopps.isEmpty()){
+        if (!reinsertopps.isEmpty()) {
             //clear existing open opps and recreate
-            delete [select id from Opportunity where npe03__Recurring_Donation__c IN :reinsertopps and isClosed = false];
-            if (reinsertopps.size() == 1 || system.isFuture() || system.isBatch()){
+            delete [SELECT Id FROM Opportunity WHERE npe03__Recurring_Donation__c IN :reinsertopps AND isClosed = false];
```

Comment:
I would leave this code as is since changes already made are for the coding style only. Making more changes like the suggested one can be done another time.